docs/data-collection.md

```diff
@@ -84,6 +84,24 @@ For the OpenShift 4 Developer Preview we will be sending back these exact attrib
   // subscription_sync_total is the number of times an OLM operator
   // Subscription has been synced, labelled by name and installed csv
   '{__name__="subscription_sync_total"}',
+  //
+  //OCS metrics to be collected:
+  // ceph_cluster_total_bytes gives the size of ceph cluster in bytes.
+  '{__name__="ceph_cluster_total_bytes"}',
```

Comment:
Naming for summed metrics normally follow the convention of `...units_total`  Is this a metric collected directly from ceph or a recording rule? In the latter case, could we stick to that here as well?

In the former case, what is the cadinality of this metric?

Reply:
This is not a recording rule. This is a metric directly collected from ceph. This has cardinality 1.

You can verify this in screenshot 2 of https://github.com/openshift/telemeter/pull/192#issuecomment-514568851